### PR TITLE
TR-2288 Update Injection Best Practices to cover DNS and load balancers

### PR DESCRIPTION
**Pull request by rdawemsys**
…connection, connection durations

### Support Docs Update Checklist

Note: when you open a pull request, your changes are published to https://staging.sparkpost.com/docs.

Please take a moment to check each of these steps for your pull request:

- [x] Give your pull request a meaningful name.
- [x] Include a description of your changes in the pull request.
- [x] _SparkPost staff only_: Check that your article appears [here](https://staging.sparkpost.com/docs) and looks correct.
- [x] Check the links in your article.
- [x] Use lowercase filenames.

### CODE REVIEW

File: support/tech-resources/smtp-rest-api-performance.md [hunk 1]
```diff
@@ -9,13 +9,16 @@ SparkPost wants to make sure all of our customers have the best sending experien
 
 We recommend that you send via the Transmission REST API, if possible. Overall the REST API is more efficient than SMTP and requires less resources on the client side.
 
-* Keep the size of the transmission to approximately 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
+* On timeout or any 5XX error the best practice is to retry.
```
Comment (mccammon) [inline support/tech-resources/smtp-rest-api-performance.md]: Do we recommend a wait period before the retry?

File: support/tech-resources/smtp-rest-api-performance.md [hunk 2]
```diff
@@ -9,13 +9,16 @@ SparkPost wants to make sure all of our customers have the best sending experien
...
+* Set API timeout to 300 seconds.
+* There is a 300MB limit on payload size for each transmission, but we recommend keeping a payload size no greater than 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
```
Comment (mccammon) [inline support/tech-resources/smtp-rest-api-performance.md]: Tempted to also explicitly state this includes recipients (thinking multi-recip flow).  What do you think?

Comment (rdawemsys) [reply support/tech-resources/smtp-rest-api-performance.md]: Good idea, will do

File: support/tech-resources/smtp-rest-api-performance.md (hunk 1 above)
Comment (rdawemsys) [reply support/tech-resources/smtp-rest-api-performance.md]: I'm thinking at least 5 seconds, with exponential backoff. I pushed a change with that.

File: support/tech-resources/smtp-rest-api-performance.md [hunk 3]
```diff
@@ -9,13 +9,19 @@ SparkPost wants to make sure all of our customers have the best sending experien
 
 We recommend that you send via the Transmission REST API, if possible. Overall the REST API is more efficient than SMTP and requires less resources on the client side.
 
-* Keep the size of the transmission to approximately 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
+* On timeout or any 5XX error the best practice is to retry.
+* When retrying, leave at least 5 seconds between retries. We also recommend using exponential backoff.
```
Comment (rdawemsys) [inline support/tech-resources/smtp-rest-api-performance.md]: @mostfunkyduck had this feedback on Slack, which I agree with and need to incorporate:

> 5 seconds between retries with exponential backoff sounds contradictory, should be 5 seconds for the initial retry, followed by an exponential backoff shouldn't it?

**Pull request merged**